api/handlers.go

```diff
@@ -64,7 +64,7 @@ func GetByIDSessionHandlerFunc(cache Cache, getVarsFunc GetVarsFunc) http.Handle
 
 		s, err := cache.GetByID(ID)
 		if err != nil {
-			writeErrorResponse(ctx, w, err.Error(), err, getErrorStatus(err))
+			writeErrorResponse(ctx, w, err.Error(), err, http.StatusNotFound)
```

Comment:
I don't think this is correct as this function can return an error for a various reasons not always "not found".